test/codegen/end_to_end_test.dart

```diff
@@ -121,6 +121,10 @@ void main(List<String> args) {
     runQuery('K', new QueryOptions(
         includeInherited: true, withAnnotations: [lib.getType('AnnotC')]));
 
+    runQuery('J', new QueryOptions(includeMethods: true));
+    runQuery('J2', new QueryOptions(
```

Comment:
Could you add a TODO for implementing support for excludeMethods in runQuery (or add support for it if it's as easy :-))? (it's not required because the current runQuery includes more results than will be needed, but just to keep the APIs codegen consistent and to a minimum.)